odl/test/operator/pspace_ops_test.py

```diff
@@ -10,35 +10,48 @@
 import pytest
 
 import odl
-from odl.util.testutils import all_almost_equal
+from odl.util.testutils import all_almost_equal, simple_fixture
 
 
-def test_pspace_op_init():
-    r3 = odl.rn(3)
-    I = odl.IdentityOperator(r3)
+base_op = simple_fixture(
+    'base_op',
+    [odl.IdentityOperator(odl.rn(3)),
+     odl.BroadcastOperator(odl.IdentityOperator(odl.rn(3)), 2),
+     odl.ReductionOperator(odl.IdentityOperator(odl.rn(3)), 2),
+     odl.DiagonalOperator(odl.IdentityOperator(odl.rn(3)), 2),
+     ],
```

Comment:
What does `ProductSpaceOperator(ProductSpaceOperator)` do?

Reply:
That's a very good question. It fails since `pspace_op[0]` is a `ReductionOperator`, and that is caught in the conversion function. IMO that's OK.